components/Loader.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
should this be moved in the `Shared` folder?

Reply:
Yes.